### PR TITLE
Cfg Process + Repair fix

### PR DESCRIPTION
**Pull request by BoGuu**
Self explanatory, moves the processing to a config.

I've tested this around a month ago, it's been sat around since, I'm 99% sure it's fine, but I encourage testing please.

Layout is screwed but it's fine on my end, can be updated later.

Scroll Text is there for future implementation of a VERY LARGE framework overhaul (:+1:)

Fixed: Non damaged vehicles could be repaired (as when damage _veh == 0, the car is undamaged, see https://community.bistudio.com/wiki/damage)

### CODE REVIEW

File: Altis_Life.Altis/core/vehicle/fn_vInteractionMenu.sqf [hunk 1]
```diff
@@ -38,7 +38,7 @@ _dlcVehicles = ["C_Kart_01_Blu_F","C_Kart_01_Red_F","C_Kart_01_Fuel_F","C_Kart_0
 _Btn1 ctrlSetText localize "STR_vInAct_Repair";
 _Btn1 buttonSetAction "[life_vInact_curTarget] spawn life_fnc_repairTruck;";
 
-if((life_inv_toolkit >= 1) && (damage _curTarget < 1)) then {_Btn1 ctrlEnable true;} else {_Btn1 ctrlEnable false;};
+if((life_inv_toolkit >= 1) && (damage _curTarget < 1) && (damage _curTarget > 0)) then {_Btn1 ctrlEnable true;} else {_Btn1 ctrlEnable false;};
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/vehicle/fn_vInteractionMenu.sqf]: I'd suggest just changing the exsisting ``(damage _curTarget < 1)`` to ``(damage _curTarget > 1)``

**Comment by NiiRoZz** [comment]
Just add please multiprocess exemple please for anyone want multiprocess have exemple

**Comment by tkcjesse** [comment]
@NiiRoZz  He has an example??? It's in the comments on lines 8-17...

**Comment by NiiRoZz** [comment]
Ho yes sorry i'm on my phone

**Pull request merged**